<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sniffer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_report">
       <attribute name="title">
        <string>Report</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_report">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_report"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_packet">
       <attribute name="title">
        <string>Packet</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_packet">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_packet"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
